app/assets/javascripts/components/restaurantList.js.jsx

```diff
@@ -46,21 +51,30 @@ class RestaurantList extends React.Component {
     });
     return (
       <div>
-        <span>Filter Results:</span>
-        <input type="text" placeholder="Filter by name"
-               onChange={this.handleNameFilterChanged.bind(this)} className="filter"/>
-        <div className="filter">
-          <label>Max delivery time: {this.state.filterMaxDelivery}</label>
-          <input type="range" value={this.state.filterMaxDelivery} min={this.getMinDelivery()} max={this.getMaxDelivery()}
-                 onChange={this.handleMaxDeliveryChange.bind(this)}/>
-        </div>
-        <div className="filter">
-          <label>Accepts 10Bis?</label>
-          <input type="checkbox" checked={this.state.filter10Bis} onChange={this.handle10BisChange.bind(this)}/>
+        <div id="filter-row">
+          <span>Filter Results:</span>
+          <input type="text" placeholder="Filter by name"
+                   onChange={this.handleNameFilterChanged.bind(this)} className="filter"/>
+          <div className="filter">
+            <label>Rating: {this.state.filterRating}+ stars</label>
+            <input type="range" value={this.state.filterRating} min={0} max={3}
+                   onChange={this.handleRatingChange.bind(this)}/>
```

Comment:
You can try this to avoid binding every method:

http://stackoverflow.com/questions/32192682/react-js-es6-avoid-binding-this-to-every-method

config/application.rb
```
config.react.jsx_transform_options = {
  optional: ['es7.classProperties']
}
```